aikau/src/main/resources/webscript-libs/doclib/doclib.lib.js

```diff
@@ -577,7 +577,8 @@ function getDocLibTree(options) {
                   siteId: options.siteId,
                   containerId: options.containerId,
                   rootNode: options.rootNode,
-                  rootLabel: options.rootLabel || "documentlibrary.root.label"
+                  rootLabel: options.rootLabel || "documentlibrary.root.label",
+                  useHash: (options.useHash !== false)
```

Comment:
Does the `options` object always have a `useHash` property? Otherwise, it could be `undefined` and it would pass through `true`? Or is that the intention? How about the string `"false"`?